### ISSUE TITLE
no-annex plugin --help has Python docstring leftover of Parameters

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#4520]
```
$> datalad no-annex --help
Usage: datalad no-annex [-h] [-d DATASET] [--pattern PATTERN [PATTERN ...]]
                        [--ref-dir REF_DIR] [--makedirs]

...

Parameters
----------
ref_dir : str, optional
makedirs : bool, optional

*Options*
  -h, --help, --help-np
                        show this help message. --help-np forcefully disables
                        the use of a pager for displaying the help message
  -d DATASET, --dataset DATASET
                        "specify the dataset to configure. If no dataset is
                        given, an attempt is made to identify the dataset
                        based on the current working directory. Constraints:
                        Value must be a Dataset or a valid identifier of a
                        Dataset (e.g. a path)
  --pattern PATTERN [PATTERN ...]
                        list of path patterns. Any content whose path is
                        matching any pattern will not be annexed when added to
                        a dataset, but instead will be tracked directly in
                        Git. Path pattern have to be relative to the directory
                        given by the REF_DIR option. By default, patterns
                        should be relative to the root of the dataset.
  --ref-dir REF_DIR     Relative path (within the dataset) to the directory
                        that is to be configured. All patterns are interpreted
                        relative to this path, and configuration is written to
                        a .GITATTRIBUTES file in this directory. [Default:
                        '.']
  --makedirs            If set, any missing directories will be created in
                        order to be able to place a file into --REF-DIR.

$> datalad --version
datalad 0.13.0rc1.dev13
```

didn't spot anything similar in `wtf` but didn't check all other plugins.
Might get solved depending on the answer/decision on #4519 (dissolve plugins)

**Comment by kyleam** [comment]
Isn't this simply because `no_annex` oddly puts a parameters section in its class docstring?

https://github.com/datalad/datalad/blob/7b61629d6e84168a346a7e2b94bcb50657468fe7/datalad/plugin/no_annex.py#L38-L42

I'd guess that f892cdf69 (RF: Convert no_annex plugin to a command, 2018-03-02) just left those in by mistake.

**Comment by yarikoptic** [comment]
ha - indeed. I should have just looked into the code.  I will just strip it and push directly to `maint` to close this one

**Comment by yarikoptic** [comment]
closed by pushed f03c67ae273b831fee4f135ccaab37776f6ed632 which forgot Closes
